tecken/apps.py

```diff
@@ -19,3 +20,15 @@ def ready(self):
         # The app is now ready.
 
         markus.configure(settings.MARKUS_BACKENDS)
+
+        connection = get_redis_connection('store')
+        maxmemory_policy = connection.info()['maxmemory_policy']
+        if maxmemory_policy != 'allkeys-lru':
+            logger.warning(
+                "The 'store' Redis cache was not configured to be an "
+                "LRU cache because the maxmemory-policy was set to '{}'. "
+                "Now changed to 'allkeys-lru'.".format(
```

Comment:
Here I wouldn't state, that it has already been changed when the actual change only happens in the next line which may still raise an error if the Redis server doesn't accept a config set call. IIRC some Redis hosters don't allow this type of change in favor of web ui config changes.